### PR TITLE
Home page layouts for remaining Pennwell sites

### PR DESCRIPTION
**Pull request by brandonbk**

### CODE REVIEW

File: sites/intelligent-aerospace/server/templates/index.marko [hunk 1]
```diff
@@ -34,25 +34,79 @@ $ const adSlots = {
...
+        native-x={ placement: 'list1', aliases: ['unmanned'], index: 2 }
+      />
+    </div>
+  </div>
   <div class="row">
     <div class="col-lg-4 mb-block">
       <theme-pennwell-published-content-query-list
         query={
-          contentTypes: ["Video"],
+          contentTypes: ["Video", "Webinar"],
```
Comment (zarathustra323) [inline sites/intelligent-aerospace/server/templates/index.marko]: I'm assuming you don't want to include Webinars with Videos?

File: sites/militaryaerospace/server/templates/index.marko [hunk 2]
```diff
@@ -49,35 +78,98 @@ $ const adSlots = {
...
+        }
+        header={ title: "Videos", href: "/videos" }
+      />
+    </div>
+    <div class="col-lg-4 mb-block">
+      <endeavor-content-query-section-list
+        limit=5
+        skip=5
+        section-alias=section.alias
+        header={ title: "More from Vision Systems Design" }
```
Comment (zarathustra323) [inline sites/militaryaerospace/server/templates/index.marko]: This should say Military Aerospace

Update this to use:
```js
header={ title: `More from ${out.global.config.siteName()}` }
```

File: sites/lightwaveonline/server/templates/index.marko [hunk 3]
```diff
@@ -71,19 +127,41 @@ $ const adSlots = {
...
+        }
+        header={ title: "Videos", href: "/videos" }
+      />
+    </div>
+    <div class="col-lg-8 mb-block">
+      <endeavor-content-query-section-list
+        limit=5
+        skip=5
+        section-alias=section.alias
+        header={ title: "More from Vision Systems Design" }
```
Comment (zarathustra323) [inline sites/lightwaveonline/server/templates/index.marko]: This should say Lightwave Online...

Update this to use:
```js
header={ title: `More from ${out.global.config.siteName()}` }
```

File: sites/bioopticsworld/server/templates/index.marko [hunk 4]
```diff
@@ -49,35 +78,70 @@ $ const adSlots = {
...
+        }
+        header={ title: "Videos", href: "/videos" }
+      />
+    </div>
+    <div class="col-lg-4 mb-block">
+      <endeavor-content-query-section-list
+        limit=4
+        skip=5
+        section-alias=section.alias
+        header={ title: "More from BioOptics World" }
```
Comment (zarathustra323) [inline sites/bioopticsworld/server/templates/index.marko]: Update this to use:
```js
header={ title: `More from ${out.global.config.siteName()}` }
```

File: sites/cablinginstall/server/templates/index.marko [hunk 5]
```diff
@@ -49,35 +78,97 @@ $ const adSlots = {
...
+        header={ title: "Videos", href: "/videos" }
       />
     </div>
     <div class="col-lg-4 mb-block">
-      <endeavor-magazine-query-latest-issue publication-id="5ca3d82c75a25456040041a9" header="Current Issue" />
+      <endeavor-content-query-section-list
+        limit=4
+        skip=5
+        section-alias=section.alias
+        header={ title: "More from Cabling Installation" }
```
Comment (zarathustra323) [inline sites/cablinginstall/server/templates/index.marko]: Update this to use:
```js
header={ title: `More from ${out.global.config.siteName()}` }
```

File: sites/intelligent-aerospace/server/templates/index.marko [hunk 6]
```diff
@@ -62,22 +116,27 @@ $ const adSlots = {
...
+        header={ title: "White Papers & Webcasts", href: "/white-papers" }
       />
     </div>
     <div class="col-lg-4 mb-block">
-      <endeavor-magazine-query-latest-issue publication-id="5ca4da3c75a2546b050041a9" header="Current Issue" />
+      <endeavor-content-query-section-list
+        limit=4
+        skip=5
+        section-alias=section.alias
+        header={ title: "More from Intelligent Aerospace" }
```
Comment (zarathustra323) [inline sites/intelligent-aerospace/server/templates/index.marko]: Update this to use:
```js
header={ title: `More from ${out.global.config.siteName()}` }
```

File: sites/laserfocusworld/server/templates/index.marko [hunk 7]
```diff
@@ -120,11 +120,11 @@ $ const adSlots = {
...
     <div class="col-lg-4 mb-block">
-      <endeavor-content-query-feed
-        header="More from LaserFocusWorld"
-        limit=7
+      <endeavor-content-query-section-list
+        limit=4
         skip=5
-        section-id=section.id
+        section-alias=section.alias
+        header={ title: "More from Laser Focus World" }
```
Comment (zarathustra323) [inline sites/laserfocusworld/server/templates/index.marko]: Update this to use:
```js
header={ title: `More from ${out.global.config.siteName()}` }
```

File: sites/rdhmag/server/templates/index.marko [hunk 8]
```diff
@@ -34,43 +34,86 @@ $ const adSlots = {
...
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col-lg-4 mb-block">
+      <endeavor-content-query-section-list
+        limit=5
+        section-alias="career-profession"
+        header={ title: "Career & Profession", href: "/career-profession" }
+        native-x={ placement: 'list1', aliases: ['career\-profession'], index: 4 }
```
Comment (zarathustra323) [inline sites/rdhmag/server/templates/index.marko]: No `\` in the aliases

File: sites/utilityproducts/server/templates/index.marko [hunk 9]
```diff
@@ -33,21 +33,77 @@ $ const adSlots = {
...
+        header={ title: "Test & Measurement", href: "/test-measurement" }
+        native-x={ placement: 'list1', aliases: ['test-measurement'], index: 2 }
+      />
+    </div>
+    <div class="col-lg-4 mb-block">
+      <endeavor-content-query-section-list
+        limit=3
+        skip=5
+        section-alias=section.alias
+        header={ title: "More from Utility Products" }
```
Comment (zarathustra323) [inline sites/utilityproducts/server/templates/index.marko]: Update this to use:
```js
header={ title: `More from ${out.global.config.siteName()}` }
```

File: sites/vision-systems/server/templates/index.marko [hunk 10]
```diff
@@ -49,35 +79,97 @@ $ const adSlots = {
...
+        }
+        header={ title: "Videos", href: "/videos" }
+      />
+    </div>
+    <div class="col-lg-4 mb-block">
+      <endeavor-content-query-section-list
+        limit=4
+        skip=5
+        section-alias=section.alias
+        header={ title: "More from Vision Systems Design" }
```
Comment (zarathustra323) [inline sites/vision-systems/server/templates/index.marko]: Update this to use:
```js
header={ title: `More from ${out.global.config.siteName()}` }
```

File: sites/broadbandtechreport/server/templates/index.marko [hunk 11]
```diff
@@ -50,32 +79,57 @@ $ const adSlots = {
...
-      <theme-pennwell-published-content-query-list
-        query={
-          contentTypes: ["Video"],
-          limit: 4,
-        }
-        header={ title: "Videos", href: "/videos" }
+      <endeavor-content-query-section-list
+        limit=3
+        section-alias="video"
+        header={ title: "Video", href: "/video" }
```
[the remark this answers is not in the document]
Comment (brandonbk) [reply sites/broadbandtechreport/server/templates/index.marko]: Yep, they do; Video is a topic for them.

File: sites/intelligent-aerospace/server/templates/index.marko (hunk 1 above)
Comment (brandonbk) [reply sites/intelligent-aerospace/server/templates/index.marko]: Removed, good catch.

**Comment by brandonbk** [comment]
@zarathustra323 all cleaned up.

**Pull request merged**